docs/getting-started/provision-infrastructure.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Provision Infrastructure
+toc: true
+weight: 3
+indent: true
+---
+
+# Provision Infrastructure
+
+Crossplane's core feature is provisioning infrastructure anywhere using the
+Kubernetes API. Once you have [installed a provider] and [configured your
```

Comment:
I wonder how does this statement work with the adoption of an application model such as OAM in crossplane?

Reply:
I suspect this might be meaning core in the foundational/fundamental/lowest level sense. We'll update the wording to ensure equal footing is given to application models.

Reply:
Definitely, I updated it to say "Crossplane allows you to provision infrastructure anywhere using the Kubernetes API." since that is what this section is about :+1: